src/emscripten-optimizer/parser.cpp

```diff
@@ -14,101 +14,103 @@
  * limitations under the License.
  */
 
+#include <unordered_map>
+
 #include "parser.h"
 
 namespace cashew {
 
 // common strings
 
-IString TOPLEVEL("toplevel");
-IString DEFUN("defun");
-IString BLOCK("block");
-IString VAR("var");
-IString CONST("const");
-IString CONDITIONAL("conditional");
-IString BINARY("binary");
-IString RETURN("return");
-IString IF("if");
-IString ELSE("else");
-IString WHILE("while");
-IString DO("do");
-IString FOR("for");
-IString SEQ("seq");
-IString SUB("sub");
-IString CALL("call");
-IString LABEL("label");
-IString BREAK("break");
-IString CONTINUE("continue");
-IString SWITCH("switch");
-IString STRING("string");
-IString TRY("try");
-IString INF("inf");
-IString NaN("nan");
-IString LLVM_CTTZ_I32("_llvm_cttz_i32");
-IString UDIVMODDI4("___udivmoddi4");
-IString UNARY_PREFIX("unary-prefix");
-IString UNARY_POSTFIX("unary-postfix");
-IString MATH_FROUND("Math_fround");
-IString MATH_CLZ32("Math_clz32");
-IString INT64("i64");
-IString INT64_CONST("i64_const");
-IString SIMD_FLOAT32X4("SIMD_Float32x4");
-IString SIMD_FLOAT64X2("SIMD_Float64x2");
-IString SIMD_INT8X16("SIMD_Int8x16");
-IString SIMD_INT16X8("SIMD_Int16x8");
-IString SIMD_INT32X4("SIMD_Int32x4");
-IString PLUS("+");
-IString MINUS("-");
-IString OR("|");
-IString AND("&");
-IString XOR("^");
-IString L_NOT("!");
-IString B_NOT("~");
-IString LT("<");
-IString GE(">=");
-IString LE("<=");
-IString GT(">");
-IString EQ("==");
-IString NE("!=");
-IString DIV("/");
-IString MOD("%");
-IString MUL("*");
-IString RSHIFT(">>");
-IString LSHIFT("<<");
-IString TRSHIFT(">>>");
-IString HEAP8("HEAP8");
-IString HEAP16("HEAP16");
-IString HEAP32("HEAP32");
-IString HEAPF32("HEAPF32");
-IString HEAPU8("HEAPU8");
-IString HEAPU16("HEAPU16");
-IString HEAPU32("HEAPU32");
-IString HEAPF64("HEAPF64");
-IString F0("f0");
-IString EMPTY("");
-IString FUNCTION("function");
-IString OPEN_PAREN("(");
-IString OPEN_BRACE("[");
-IString OPEN_CURLY("{");
-IString CLOSE_CURLY("}");
-IString COMMA(",");
-IString QUESTION("?");
-IString COLON(":");
-IString CASE("case");
-IString DEFAULT("default");
-IString DOT("dot");
-IString PERIOD(".");
-IString NEW("new");
-IString ARRAY("array");
-IString OBJECT("object");
-IString THROW("throw");
-IString SET("=");
-IString ATOMICS("Atomics");
-IString COMPARE_EXCHANGE("compareExchange");
-IString LOAD("load");
-IString STORE("store");
-IString GETTER("get");
-IString SETTER("set");
+wasm::IString TOPLEVEL("toplevel");
```

Comment:
Perhaps these should no longer belong in the `cashew` namespace if they are no longer cashew strings?